django/core/serializers/python.py

```diff
@@ -75,42 +75,64 @@ def getvalue(self):
         return self.objects
 
 
-def Deserializer(object_list, *, using=DEFAULT_DB_ALIAS, ignorenonexistent=False, **options):
+class Deserializer(base.Deserializer):
     """
     Deserialize simple Python objects back into Django ORM instances.
-
     It's expected that you pass the Python objects themselves (instead of a
     stream or a string) to the constructor
     """
-    handle_forward_references = options.pop('handle_forward_references', False)
-    field_names_cache = {}  # Model: <list of field_names>
-
-    for d in object_list:
+    def __init__(self, object_list, *, using=DEFAULT_DB_ALIAS, ignorenonexistent=False, **options):
+        super().__init__(stream_or_string=object_list, **options)
```

Comment:
I don't think we want to subclass `base.Deserializer`. Instead, we can just do `self.object_list = object_list` and use that instead of `self.stream` below.

Reply:
Super, I will work on this change.